### DISCUSSION TITLE
Do I need this if I'm using sentry/vue?

### DISCUSSION THREAD

**Discussion by alexcroox** [getsentry/sentry-capacitor#723] [Q&A]
I've been using Vue with Capacitor for a number of years, and using the Sentry Vue/Browser SDKs, which work fine both on the web and inside Capacitor.

Would I need to stop using it if I used this SDK? What would be the advantage? Would I get duplicate error reports if I used both?

**Comment by boozedog** [comment]
I'm wondering same thing myself, esp since this package isn't compatible with most recent @sentry/vue
